src/command_modules/azure-cli-vm/setup.py

```diff
@@ -47,6 +47,7 @@
         'azure',
         'azure.cli',
         'azure.cli.command_modules',
+        'Paramiko'
```

Comment:
This should be in the dependencies section `Paramiko`.
Also, should be lower case P?